subsys/bluetooth/controller/ll_sw/ull_adv_sync.c

```diff
@@ -225,7 +225,7 @@ uint8_t ll_adv_sync_param_set(uint8_t handle, uint16_t interval, uint16_t flags)
 }
 
 #if defined(CONFIG_BT_CTLR_ADV_ISO) && defined(CONFIG_BT_TICKER_EXT_EXPIRE_INFO)
-void ull_adv_iso_created(struct ll_adv_sync_set *sync)
+void ull_adv_sync_iso_created(struct ll_adv_sync_set *sync)
```

Comment:
```suggestion
void ull_adv_sync_iso_created(const struct ll_adv_sync_set *sync)
```

Reply:
done.

Reply:
Had to remove the `const` as `sync` passed as ticker callback parameter which can modify the `sync` object/memory.